src/style.css

```
@@ -1 +1,89 @@
-/* Styles go here */
+* {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: block;
+}
+
+.image {
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  list-style-type: none;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  flex-direction: row;
+  padding: 0;
+
+  height: 60px;
+}
+
+.nav.item {
+  display: inline-block;
+  height: 60px;
+  margin: 10px;
+}
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  justify-content: end;
+  position: relative;
+  height: 60px;
+  text-decoration: none;
+  color: #000;
+  margin: 0 0 0 20px;
+  transition: 0.3s;
```

Потрібно вказувати для чого застосовуєш transition
```suggestion
  transition: color 0.3s;
```